### DISCUSSION TITLE
color_temp control

### DISCUSSION THREAD

**Discussion by TAW-V** [basnijholt/adaptive-lighting#849] [Q&A]
Hi, 

This are my settings:
- sunset time fixed at 20:10
- brightness_mode_time_dark: 9000
- brightness_mode_time_light: 0
- brightness_mode: linear

This means that the minimal brightness value will be reached at 22:40.
I'm enabling adaptive-lighting at 18:00

How do I control the transition to  min_color_temperature?
Currently it transitions very rapidly to from max_color_temperature to min_color_temperature.

Ideally, it should transition similarly as the brightness.

**Comment by smoores-dev** [comment]
I think I have the same question, or at least a similar one. It seems that AL completely adapts the color temperature before it starts adapting the brightness, and it does so quite quickly. Ideally, I think, the temperature and brightness transitions would occur simultaneously!
